src/tokens.ts

```diff
@@ -183,6 +185,7 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.OPTIMISM_SEPOLIA]: "0x5fd84259d66Cd46123540766Be93DFE6D43130D7",
       [CHAIN_IDs.POLYGON]: "0x3c499c542cEF5E3811e1192ce70d8cC03d5c3359",
       [CHAIN_IDs.POLYGON_AMOY]: "0x41E94Eb019C0762f9Bfcf9Fb1E58725BfB0e7582",
+      [CHAIN_IDs.SCROLL]: "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4",
```

Comment:
nb. This is technically the bridged USDC contract that's deployed by the Scroll team, but it's the only USDC variant deployed to Scroll and they're still calling it USDC. I'm making a mental note that we might need to relocate it to the USDC.e definitions, though.